src/CMakeLists.txt

```diff
@@ -191,10 +191,12 @@ if (NOT MSVC)
     endif()
 
     if (CYGWIN OR MINGW OR MSYS)
-        target_link_libraries( exiv2lib PRIVATE  psapi ws2_32 )
+        target_link_libraries( exiv2lib PRIVATE psapi ws2_32 )
     endif()
 
     target_link_libraries( exiv2lib PRIVATE Threads::Threads)
+else()
+    target_link_libraries( exiv2lib PRIVATE psapi ws2_32 )
```

Comment:
Now we have `target_link_libraries( exiv2lib PRIVATE psapi ws2_32 )` in two different places (when **MSVC** or when `CYGWIN OR MINGW OR MSYS`). I think that it would be simpler to just have it once when `WIN32`.

Reply:
We could reuse for that the already existing conditional in line 184.